tests/Model/StubsContainer.php

```diff
@@ -59,7 +59,13 @@ public function getFunctions(): array
     public function addFunction(PHPFunction $function): void
     {
         if (isset($function->name)) {
-            $this->functions[$function->name] = $function;
+            if (array_key_exists($function->name, $this->functions)) {
+                $amount = count(array_filter($this->functions,
+                    fn(PHPFunction $nextFunction) => $nextFunction->name === $function->name));
+                $this->functions[$function->name . '_duplicated_' . $amount] = $function;
```

Comment:
Hmmm... I don't like the approach of saving a number of duplicates in the array key. I believe it's better to create a property inside `PHPFunction`.

Reply:
It was done intentionally to store all duplicates in `StubsContainer`. If don't use unique keys duplicate properties will overwrite previous property

Reply:
Oh, I get it, you store different functions with a unique id. I thought that the information about the total number of duplicates is encrypted in a key name.